src/liballoc/vec.rs

```diff
@@ -748,28 +748,64 @@ impl<T> Vec<T> {
         self
     }
 
-    /// Sets the length of a vector.
+    /// Forces the length of a vector to a particular value.
```

Comment:
```suggestion
    /// Forces the length of a vector to be the given value in `new_len`.
```
particular doesn't sound very particular. =P

Reply:
> Forces the length of the vector to `new_len`

seems less mouthful to me.

Reply:
Updated the suggestion to @nagisa's wording which is more direct / better.